### ISSUE TITLE
JDBC dbtable query option support

### ISSUE THREAD

**Issue by DaimonPl** [AbsaOSS/spline-spark-agent#99]
That's tricky one :) I've found one case where JDBC datasoruce is used for reading with `dbtable` option, where `dbtable` is a query :)

here's example what spline reports in such situation

```
jdbc:mysql://host.net:3336/my_database?serverTimezone=Poland&useSSL=false&characterEncoding=utf8:( SELECT *\n  FROM some_table\n  WHERE x_date = CAST('2020-06-14' AS DATE)\n    AND y IN (SELECT y FROM other_table)\n    AND y BETWEEN 26409 AND 675433\n) tmp
```

It probably would be very difficult but ideally 2 lineage sources `some_table` and `other_table` could be reported here

**Comment by wajda** [comment]
Yeah... just like with the #69 the rabbit hole actually goes much deeper. `some_table` could easily be a view putting us in the same situation as with an inlined query, but this time we wouldn't even see the query.

**Comment by wajda** [comment]
AbsaOSS/spline#689
